tests/unit/amo/components/TestHeader.js

```diff
@@ -98,6 +98,7 @@ describe(__filename, () => {
     const link = wrapper.find('.Header-user-menu-collections-link');
 
     expect(link).toHaveLength(1);
+    expect(link.prop('href')).toEqual('/collections/');
```

Comment:
So this test will fail as soon as we'll turn the feature on. Is that expected? Maybe we could have tests with `getFakeConfig()`: https://github.com/mozilla/addons-frontend/blob/d9b067d4a53c25821d6defc2eb75583ac4892f1f/tests/unit/helpers.js#L459

Reply:
The `Header` component does not currently allow us to inject a `config` object, so we can't use `getFakeConfig()` without changing that. I see that this issue was avoided for the View Profile and Edit Profile links by simply not asserting anything about the href/to property. Because these flags are temporary, maybe it's fine to just do the same for the collections like, rather than going through the trouble of updating the `Header` component and creating tests based on the value of the flag?

Or, if you think we should enhance all of the tests to assert on the link href/to, then maybe that can be a follow-up?

For now (pending your response), I'll just remove that assertion so the test won't fail when the flag changes.